python/ray/train/_internal/dataset_spec.py

```diff
@@ -28,8 +28,8 @@ class RayDatasetSpec:
         training workers (to use as locality hints). The Callable is expected to
         return a list of RayDatasets or a list of dictionaries of RayDatasets,
         with the length of the list equal to the length of the list of actor handles.
-        If None is provided, the provided Ray Dataset(s) will be simply be split using
-        the actor handles as locality hints.
+        If None is provided, the provided Ray Dataset(s) will be simply equally
+        splitted without locality hints.
```

Comment:
```suggestion
        split.
```

Do we still want to explicitly say no locality hints?